cluster/manifests/01-admission-control/config.yaml

```diff
@@ -113,8 +113,7 @@ data:
   node.node-not-ready-taint.enable: "{{ .Cluster.ConfigItems.teapot_admission_controller_node_not_ready_taint }}"
   node.extended-node-restriction.enable: "true"
 
-  # keep legacy configs during rollouts
-  pod.node-lifecycle.provider: "{{ .Cluster.ConfigItems.teapot_admission_controller_node_lifecycle_provider }}"
+  pod.node-lifecycle.provider: "zalando"
```

Comment:
This setting is replaced by the per node pool setting below. There is no longer a global setting but every node pool, including the default, gets a setting.

Reply:
I dropped it.